README.md

```diff
@@ -15,24 +15,45 @@ Access to both documents is available to members of the
 [kubernetes-sig-multicluster google
 group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
 
+# Concepts
+
 <p align="center"><img src="docs/images/concepts.png" width="711"></p>
 
-# Concepts
+Federation is configured with two types of information:
+
+- **Type configuration** declares which API types federation should handle
```

Comment:
Period to the end, ditto for others.

How about s/Type configuration/Type Configuration to keep consistent with the diagram?

Reply:
Got it, thanks @pmorie